Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -117,6 +117,10 @@
                         }
                     </TableRow>
                 }
+                @if ( HasAggregates && !Virtualize && ( AggregateRowPosition == DataGridAggregateRowPosition.Top || AggregateRowPosition == DataGridAggregateRowPosition.TopAndBottom ) )

```

Comment:
Maybe also have a property getter to check to for AggregateRowPosition as we did for most other stuff? 

eg. `bool AggregateOnTop => AggregateRowPosition == DataGridAggregateRowPosition.Top || AggregateRowPosition == DataGridAggregateRowPosition.TopAndBottom;`